### DISCUSSION TITLE
How can I set the pagination at top of the tabular layout?

### DISCUSSION THREAD

**Discussion by t-kawatsu** [directus/directus#10043] [Feature Requests]
I want to set the pagination at upper side on the tabular layout.
How could I do that?

Thank you.

**Comment by benhaynes** [comment]
What do you mean by "upper side"?

**Reply by t-kawatsu** [reply]
oh, sorry. I meant that I want to put the pagination on the top of table, not just only bottom.

**Reply by benhaynes** [reply]
Ahh, there's no way to do that yet... so I'll convert this into a Feature Request.

**Reply by t-kawatsu** [reply]
I see. thank you.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
